### PR TITLE
IGAPP-285: Firebase not initializing

### PR DESCRIPTION
**Pull request by maxammann**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by maxammann** [comment]
So the fix was to include a firebase config for the test env. I explained why this happens here: https://github.com/invertase/react-native-firebase/issues/360

For this I also had to adjust the bundle ids as they were invalid under certain circumstances. I also added development certificates using fastlane match.

**Comment by maxammann** [comment]
The cool this is: you can test now push notifications with test builds.

File: native/ios/Podfile.lock [hunk 1]
```diff
@@ -11,38 +11,37 @@ PODS:
...
+    - GoogleUtilities/UserDefaults (~> 7.0)
+  - FirebaseMessaging (7.4.0):
+    - FirebaseCore (~> 7.0)
+    - FirebaseInstanceID (~> 7.0)
+    - GoogleUtilities/AppDelegateSwizzler (~> 7.0)
+    - GoogleUtilities/Environment (~> 7.0)
+    - GoogleUtilities/Reachability (~> 7.0)
+    - GoogleUtilities/UserDefaults (~> 7.0)
   - Flipper (0.54.0):
     - Flipper-Folly (~> 2.2)
```
Comment (maxammann) [inline native/ios/Podfile.lock:46]: @Znippet if you also do some upgrades, do not upgrade this one :D it makes the app crash when developing

Comment (maxammann) [reply native/ios/Podfile.lock:46]: Oh and also: can you set your first name on github? that way I can tag you more easily using @

**Review by klinzo — APPROVED** [review]

**Comment by SteffiStuffel** [comment]
looks sane, but don't know...

**Review by SteffiStuffel — APPROVED** [review]

**Pull request merged**